install/package.rst

```diff
@@ -104,6 +104,12 @@ Add Repository
                   $ echo "deb [signed-by=/etc/apt/trusted.gpg.d/pkgr-zammad.gpg] https://dl.packager.io/srv/deb/zammad/zammad/stable/debian 11 main"| \
                     tee /etc/apt/sources.list.d/zammad.list > /dev/null
 
+            Debian 12
+               .. code-block:: sh
+
+                  $ echo "deb [signed-by=/etc/apt/trusted.gpg.d/pkgr-zammad.gpg] https://dl.packager.io/srv/deb/zammad/zammad/stable/debian 12 main"| \
+                    tee /etc/apt/sources.list.d/zammad.list > /dev/null
+
```

Comment:
This looks fine, for sure, but I think all these blocks have unnecessary indentation. They are rendered under **Install Repository Key** subtitle, but they should be on the same level.

I'm fine with postponing this change for later, it just looks strange to me.

Reply:
I'll check and update, sorry about that.